### ISSUE TITLE
FOM Prod UAT Test

### ISSUE THREAD

**Issue by ArogeG** [bcgov/nr-fom#102]
Data has been moved, but not verified.  Please help!

Old TEST
https://fom-test.nrs.gov.bc.ca/api/
https://fom-test.nrs.gov.bc.ca/admin/
https://fom-test.nrs.gov.bc.ca/public/

New TEST
https://fom-test.apps.silver.devops.gov.bc.ca/api/
https://fom-test.apps.silver.devops.gov.bc.ca/admin/
https://fom-test.apps.silver.devops.gov.bc.ca/public/

New DEMO
https://fom-demo.apps.silver.devops.gov.bc.ca/api/
https://fom-demo.apps.silver.devops.gov.bc.ca/admin/
https://fom-demo.apps.silver.devops.gov.bc.ca/public/

**Comment by DerekRoberts** [comment]
Old TEST
https://fom-test.nrs.gov.bc.ca/api/
https://fom-test.nrs.gov.bc.ca/admin/
https://fom-test.nrs.gov.bc.ca/public/

New TEST
https://fom-test.apps.silver.devops.gov.bc.ca/api/
https://fom-test.apps.silver.devops.gov.bc.ca/admin/
https://fom-test.apps.silver.devops.gov.bc.ca/public/

New DEMO
https://fom-demo.apps.silver.devops.gov.bc.ca/api/
https://fom-demo.apps.silver.devops.gov.bc.ca/admin/
https://fom-demo.apps.silver.devops.gov.bc.ca/public/

**Comment by amee99073** [comment]
Tested both test and demo.
Upload caused an odd error once in test (and never again) 
Error stated that I did not have permissions to perform the action - but the file I was uploading uploaded anyway.
Tried it again with another upload file - no error
tried again with same file - no error
New session - no error
deleted all uploads, new session - no error.
Could not make error happen again.
Error was a little different from all others as the entire text box was trimmed in red. The rest of the error messages appear in pop ups with straight, white background, no coloured trim.
